resources/views/backend/user/show.blade.php

```diff
@@ -0,0 +1,39 @@
+@extends('backend.layouts.master')
+@section('content')
+    <div class="col-md-12 col-sm-12 col-xs-12 profile_left">
+
+        <div class="profile_img">
+
+            <!-- end of image cropping -->
+            <div id="crop-avatar" class="col-md-2">
+                <!-- Current avatar -->
+                <img class="img-responsive avatar-view" src="{!! config('define.image_default') !!}" alt="Avatar" title="Change the avatar">
```

Comment:
[rank C] Do we need this title of avatar image? If we use it, please put in language file.